dropwizard-app/src/test/java/org/coner/ConerDropwizardApplicationTest.java

```diff
@@ -46,41 +52,64 @@
 public class ConerDropwizardApplicationTest {
 
     // Test constants
-    private static final int NUMBER_OF_RESOURCES = 6;
+    private static final int NUMBER_OF_RESOURCES = 7;
 
     // application dependencies
     private final HibernateBundle<ConerDropwizardConfiguration> hibernate = mock(HibernateBundle.class);
     private final ConerCoreService conerCoreService = mock(ConerCoreService.class);
 
     // Event
-    private final EventBoundary eventBoundary = mock(EventBoundary.class);
-    private final EventDao eventDao = mock(EventDao.class);
-    private final EventGateway eventGateway = mock(EventGateway.class);
+    @Mock
+    EventBoundary eventBoundary;
+    @Mock
+    EventDao eventDao;
+    @Mock
+    EventGateway eventGateway;
 
     // Registration
-    private final RegistrationBoundary registrationBoundary = mock(RegistrationBoundary.class);
-    private final RegistrationDao registrationDao = mock(RegistrationDao.class);
-    private final RegistrationGateway registrationGateway = mock(RegistrationGateway.class);
+    @Mock
+    RegistrationBoundary registrationBoundary;
+    @Mock
+    RegistrationDao registrationDao;
+    @Mock
+    RegistrationGateway registrationGateway;
 
     // HandicapGroup
```

Comment:
Why not update HandicapGroup objects to use the Mock annotation while you are at it?